### PR TITLE
Add PCL and OpenCV to conda dependencies in conda-based CI

### PR DESCRIPTION
**Pull request by traversaro**
And remove them from homebrew.

### CODE REVIEW

File: bindings/python/FloatingBaseEstimators/src/LeggedOdometry.cpp [hunk 1]
```diff
@@ -58,13 +58,13 @@ void CreateLeggedOdometry(pybind11::module& module)
...
+             py::overload_cast<const InternalState&>(&FloatingBaseEstimator::resetEstimator),
              py::arg("new_state"))
         .def("reset_estimator",
              py::overload_cast<const Eigen::Quaterniond&, const Eigen::Vector3d&>(
-                 &LeggedOdometry::resetEstimator),
-             py::arg("new_imu_orientation"),
-             py::arg("new_imu_position"))
+                 &FloatingBaseEstimator::resetEstimator),
+             py::arg("new_base_orientation"),
+             py::arg("new_base_position"))
```
Comment (prashanthr05) [inline bindings/python/FloatingBaseEstimators/src/LeggedOdometry.cpp:67]: Maybe a rebase over the latest master will remove this change?

Comment (traversaro) [reply bindings/python/FloatingBaseEstimators/src/LeggedOdometry.cpp:67]: Sorry, there was a typo in the main message, this PR dependend on https://github.com/dic-iit/bipedal-locomotion-framework/pull/380, not https://github.com/dic-iit/bipedal-locomotion-framework/pull/381 as erroneously reported.

Comment (traversaro) [reply bindings/python/FloatingBaseEstimators/src/LeggedOdometry.cpp]: Anyhow, this is done now.

**Review by prashanthr05 — APPROVED** [review]

**Comment by prashanthr05** [comment]
cc @GiulioRomualdi could merge this when CI passes.

**Comment by traversaro** [comment]
I manually killed the C++ CI workflow to permit to conda jobs to run faster, nothing to be worried about if you see a bit of red.

**Comment by traversaro** [comment]
Compilation works fine with PCL 1.12 from conda, so I think we can just remove PCL from the brew build.

**Comment by traversaro** [comment]
Disabling changelog as this is a CI only change.

**Comment by traversaro** [comment]
> 
> 
> Compilation works fine with PCL 1.12 from conda-forge, so I think we can just remove PCL from the brew build.

Done in https://github.com/dic-iit/bipedal-locomotion-framework/pull/383/commits/722833981d7a8ff7c0a2f818318c07fa2482beec .

**Comment by traversaro** [comment]
Not sure why it marks the build as red.

**Comment by prashanthr05** [comment]
> Not sure why it marks the build as red.

I think since the workflow was cancelled, maybe?

**Comment by traversaro** [comment]
> 
> 
> > Not sure why it marks the build as red.
> 
> I think since the workflow was cancelled, maybe?

I re-started the CI, but then another conda regression appeared, related to https://github.com/conda-incubator/setup-miniconda/issues/186 . I added a workaround in https://github.com/dic-iit/bipedal-locomotion-framework/pull/383/commits/692808f7830ee8941c2e892f05afac422feafa1c .

**Review by GiulioRomualdi — APPROVED** [review]
Finally green. I think we can merge it. Is it OK for you @traversaro ?

**Comment by traversaro** [comment]
> 
> 
> Finally green. I think we can merge it. Is it OK for you @traversaro ?

Yes!

**Pull request merged**